repose-aggregator/core/core-service-api/src/main/resources/META-INF/schema/system-model/system-model.xsd

```diff
@@ -333,7 +333,7 @@
                    saxon:message="Must have at least one destination defined.">
         </xs:assert>
         <xs:assert vc:minVersion="1.1"
-                   test="((count(mod:endpoint)+count(mod:target)) eq 1) or (count(mod:*[xs:boolean(@default) = true()]) eq 1)"
+                   test="count(mod:*[xs:boolean(@default) = true()]) eq 1"
```

Comment:
Actually, while that was the assumption in the schema, it was not an assumption we made in code. So that mismatch in expectation led to a bug.

Now, rather than this change, we could have updated the code to assume that a single item is the default. I think in the ended I opted to go this route for the sake of clarity and consistency. This approach provides the same experience regardless of whether there is one item in the sequence or many. Also, this change was one line versus the multi-line code change the other approach would have required. So that minimizes complexity in some small way.